code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -549,6 +554,7 @@ var/global/list/synth_flesh_disguises = list()
 	SA_para_min = 0
 	SA_sleep_min = 0
 	dangerous_existence = 1
```

Comment:
and technically I'm still right, if you actually attempt to give them a disguise, they always get it :P